tests/geocoding_tests.py

```diff
@@ -15,62 +15,323 @@
 # specific language governing permissions and limitations
 # under the License.
 """Unit tests for geocoding"""
+import threading
+import time
 
+import pandas as pd
+import simplejson as json
+from sqlalchemy import Float, Integer, MetaData, String
 from sqlalchemy.engine import reflection
-from sqlalchemy_utils import table_name
+from sqlalchemy.ext.declarative import declarative_base
 
 import superset.models.core as models
-from superset import db
-from superset.connectors.sqla.models import SqlaTable
+from superset import conf, db
+from superset.connectors.sqla.models import SqlaTable, TableColumn
+from superset.exceptions import TableNotFoundException
 from superset.models.core import Database
-from superset.views import core, geocoding
+from superset.utils.geocoding_utils import GeocoderUtilMock
+from superset.views.geocoding import Geocoder
 
 from .base_tests import SupersetTestCase
 
 
 class GeocodingTests(SupersetTestCase):
-    superset = core.Superset()
+    test_database = None
+    sqla_departments = None
```

Comment:
Name could be more descriptive, this sounds like an sqla-built-in or something instead of he table we use for testing purposes